devicon.json

```
@@ -3310,6 +3310,33 @@
         "color": "#2D333A",
         "aliases": []
     },
+    {
+        "name": "socketio",
+        "tags": [
+            "library",
+            "networking",
+            "websockets"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```

For `font`, please put the SVG versions that are font-friendly here. The non-existent SVG versions should go in `aliases`.

Also, I'd recommend not using `aliases` anymore. We'll eventually phase it out in the future. We'll update the docs as well so people know our intentions.

```suggestion
                "original",
                "original-wordmark"
```